lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,83 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
-  margin: 0;
-  padding-top: 20px;
+  margin: 0 0 $nx-spacing-xl 0;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
+
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-l;
+
+  .nx-btn-bar {
+    margin: 0;
+    text-align: right;
+  }
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  .nx-alert {
+    margin: 0;
+
+    + .nx-btn-bar {
+      margin-top: 24px;
+    }
+  }
 }
 
 .nx-form-group {
-  margin-bottom: 20px;
+  margin-bottom: 6px;
 }
 
 .nx-form-row {
   @include container-horizontal;
-  margin-bottom: 24px;
+  margin-bottom: 6px;
 
-  .nx-form-group, .nx-fieldset {
+  .nx-form-group {
     display: inline-block;
-    margin-right: 24px;
+    margin-right: $nx-spacing-l;
     margin-bottom: 0;
-    vertical-align: bottom; // fixes issue where certain form types display lower
-
-    &:last-child {
-      margin-right: 0;
-    }
   }
-
-  .nx-label {
-    margin-bottom: 0;
-  }
-}
-
-.nx-btn-bar--forms {
-  border-top: $nx-border-dark;
-  padding-top: 8px;
 }
 
 // labels
 
 .nx-label, .nx-legend {
-  @include semi-bold();
+  @include bold();
 
-  color: $nx-text-color;
+  cursor: default;
   display: block;
-  font-size: $nx-font-size;
-  margin-bottom: $nx-spacing-xs;
-
-  .disabled {
-    color: $nx-grey-4;
-  }
+  padding: 0;
 
   .nx-text-input, .nx-textarea {
     display: block;
-    margin-top: $nx-spacing-xs;
   }
 }
 
-.nx-label--optional {
-  .nx-label__text::after {
+.nx-label__text, .nx-legend__text {
+  display: block;
+  margin-bottom: $nx-spacing-xs;
+}
+
+.nx-label--optional, .nx-legend--optional {
+  .nx-label__text::after, .nx-legend__text::after {
```

Comment:
Should probably have an example of `.nx-legend--optional` to prove it works as expected

Reply:
> Should probably have an example of .nx-legend--optional to prove it works as expected

Reply:
> Should probably have an example of `.nx-legend--optional` to prove it works as expected

Added here: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/9367790a17224976136ab993c8d0457bd224f746